File: server/withssafy/src/main/java/com/ssafy/withssafy/dto/user/UserAuthResponse.java
package com.ssafy.withssafy.dto.user;

import io.swagger.annotations.ApiModelProperty;
import lombok.Data;

@Data
public class UserAuthResponse {
    @ApiModelProperty(example = "유저 권한")
    private int auth;
    @ApiModelProperty(example = "유저 기본 정보")
    private UserInfo user;
}

File: application/WithSSAFY/app/src/main/java/com/ssafy/withssafy/src/network/response/UserAuthResponse.kt
package com.ssafy.withssafy.src.network.response

import com.google.gson.annotations.SerializedName
import com.ssafy.withssafy.src.dto.ClassRoom
import com.ssafy.withssafy.src.dto.User
import java.io.Serializable

data class UserAuthResponse(
    @SerializedName("auth") var auth: Int,
    @SerializedName("user") var user: User,
) : Serializable
